public/app/core/components/Page/Page.tsx

```diff
@@ -54,7 +54,7 @@ export const Page: PageType = ({
     <div className={cx(styles.wrapper, className)} {...otherProps}>
       {layout === PageLayoutType.Standard && (
         <CustomScrollbar autoHeightMin={'100%'} scrollTop={scrollTop} scrollRefCallback={scrollRef}>
-          <div className={styles.pageInner}>
+          <div className={cx(styles.pageInner, config.featureToggles.dockedMegaMenu && styles.newPageInner)}>
```

Comment:
if you wanna apply the class in this way, think this would be the more correct syntax
```suggestion
          <div className={cx(styles.pageInner, {
            [styles.newPageInner]: config.featureToggles.dockedMegaMenu
          })}>
```